### PR TITLE
Train rust dataset

### PR DESCRIPTION
**Pull request by HockJC**

### CODE REVIEW

File: .DS_Store
Comment (EYH0602) [inline .DS_Store:1]: please remove `.DS_Store`

File: marshallpierce/rust-base64
Comment (EYH0602) [inline marshallpierce/rust-base64:1]: why is this submodule needed?

File: training/.DS_Store
Comment (EYH0602) [inline training/.DS_Store:1]: please remove

File: training/data/Alexhuszagh-minimal-lexical.success.jsonl_0.txt
Comment (EYH0602) [inline training/data/Alexhuszagh-minimal-lexical.success.jsonl_0.txt:1]: these data is reproducable using dataloader, please store them in memory and don't commit them. please remove

**Review by EYH0602 — CHANGES_REQUESTED** [review]

File: training/data/Alexhuszagh-minimal-lexical.success.jsonl_0.txt
Comment (HockJC) [reply training/data/Alexhuszagh-minimal-lexical.success.jsonl_0.txt:1]: ok.

> these data is reproducable using dataloader, please store them in memory and don't commit them. please remove

File: results/checkpoint-1000/README.md
Comment (DataCorrupted) [inline results/checkpoint-1000/README.md:1]: DO NOT put results in git. This is a dirty branch now, please find the commit that introduced these files and send a new PR (or force push to this branch).

If it is something you want to note down, write it in confluence with a link to our server address.

File: training/data/test_dataset.jsonl
Comment (DataCorrupted) [inline training/data/test_dataset.jsonl:1]: I don't encourage you to put the training data in the repo as well.

**Review by DataCorrupted — CHANGES_REQUESTED** [review]

File: results/checkpoint-1000/README.md
Comment (HockJC) [reply results/checkpoint-1000/README.md:1]: OK

**Comment by EYH0602** [comment]
@HockJC please open  PR at https://github.com/SecurityLab-UCD/FuzzAugmentation for the updates.

**Pull request closed without merging**